CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+4.1.0
+-----
+* Added support for Python 3.8 - 3.11
+* Dropped Travis CI integration
+* Moved CI to Github Actions
+* Switched to using pytest to run test suite
```

Comment:
I would add a period at the end of each of these, just for consistency.

Reply:
oh, yes, I'll add those!